src/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -751,15 +751,9 @@ public override bool Equals(object? obj)
             return handle.Value == Value;
         }
 
-        public static bool operator ==(RuntimeMethodHandle left, RuntimeMethodHandle right)
-        {
-            return left.Equals(right);
-        }
+        public static bool operator ==(RuntimeMethodHandle left, RuntimeMethodHandle right) => left.Equals(right);
 
-        public static bool operator !=(RuntimeMethodHandle left, RuntimeMethodHandle right)
-        {
-            return !left.Equals(right);
-        }
+        public static bool operator !=(RuntimeMethodHandle left, RuntimeMethodHandle right) => !left.Equals(right);
 
         public bool Equals(RuntimeMethodHandle handle)
```

Comment:
it looks like you have forgot about this one?

```cs
public bool Equals(RuntimeMethodHandle handle) => handle.Value == Value;
```

Reply:
Didn't forget; I didn't globally refactor methods, only doing them opportunistically when I saw them.  There are hundreds if not thousands of methods in corelib causing the IDE to flag them.